frontend/packages/dev-console/src/components/import/devfile/DevfileStrategySection.tsx

```diff
@@ -106,9 +109,11 @@ const DevfileStrategySection: React.FC = () => {
   }, [recommendedStrategy, setFieldValue, strategies]);
 
   React.useEffect(() => {
-    importType === 'devfile' && devfile.devfilePath && handleDevfileChange();
+    (importType === 'devfile' || selectedStrategy.type === ImportStrategy.DEVFILE) &&
```

Comment:
We have an enum for the importType:

```suggestion
    (importType === ImportTypes.devfile || selectedStrategy.type === ImportStrategy.DEVFILE) &&
```